src/navmeas.rs

```diff
@@ -0,0 +1,82 @@
+use crate::{c_bindings, ephemeris::SatelliteState, signal::GnssSignal};
+use std::time::Duration;
+
+const NAV_MEAS_FLAG_CODE_VALID: u16 = 1 << 0;
+const NAV_MEAS_FLAG_MEAS_DOPPLER_VALID: u16 = 1 << 2;
+const NAV_MEAS_FLAG_CN0_VALID: u16 = 1 << 5;
+
+#[derive(Clone)]
+#[repr(transparent)]
+pub struct NavigationMeasurement(c_bindings::navigation_measurement_t);
+
+impl NavigationMeasurement {
+    pub fn new() -> Self {
+        Self::default()
+    }
```

Comment:
Is it preferable to define both a `new()` function and implement `Default`? If so then in this case I think they should  do the same thing. Is there a preference for `new()` calling `default()` or vice versa?

Reply:
@notoriaga any ideas here?

Reply:
There is actually a `new_without_default` clippy lint - https://rust-lang.github.io/rust-clippy/master/#new_without_default. My understanding is generally you should use `new` when working with values. `Default` is normally implemented to aid in generic programming, i.e. using `Default` as a trait bound

Reply:
I've flipped all of the new impls of `Default` to call `new()` instead. Thanks for the clarification